libsemigroups_cppyy/__init__.py

```diff
@@ -107,6 +107,9 @@ def cppyy_version():
 cppyy.cppdef("#define HPCOMBI_CONSTEXPR_FUN_ARGS")
 
 cppyy.include("libsemigroups/libsemigroups.hpp")
```

Comment:
I think you can fix this by doing:

~~~~
if compare_version_numbers(libsemigroups_version(), "1.1.0") and not compare_version_numbers("1.3.0", libsemigroups_version()):
~~~~